Integrations/integration-Office_EWS.yml

```diff
@@ -451,9 +455,45 @@ script:
             contextArrayName: 'Results',
             argsToContext: ['filter']
         },
+        'ews-move-items' : {
+            arrayArgName: ['item-ids','folder-identifier'],
+            tableTitle: 'EWS move items',
+            template:
+                '<?xml version="1.0" encoding="utf-8"?>'+
+                '<soap:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"' +
+                               ' xmlns:m="http://schemas.microsoft.com/exchange/services/2006/messages"' +
+                               ' xmlns:t="http://schemas.microsoft.com/exchange/services/2006/types"' +
+                               ' xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">' +
+                '<soap:Header>'+
+                    '<t:RequestServerVersion Version="%version%" />'+
+                    '%impersonation%'+
+                '</soap:Header>'+
+                '<soap:Body>' +
+                    '<m:MoveItem xmlns="http://schemas.microsoft.com/exchange/services/2006/messages" ' +
+                    'xmlns:t="http://schemas.microsoft.com/exchange/services/2006/types">' +
+                        '<m:ToFolderId>' +
+                            '%folder-identifier%' +
+                        '</m:ToFolderId>' +
+                        '<m:ItemIds>' +
+                            '%item-ids%' +
+                        '</m:ItemIds>' +
+                    '</m:MoveItem>' +
+                '</soap:Body>' +
+            '</soap:Envelope>',
+            innerPath: 'Envelope.Body.MoveItemResponse.ResponseMessages.MoveItemResponseMessage.Items.Message',
+            resStatus: 'Envelope.Body.MoveItemResponse.ResponseMessages.MoveItemResponseMessage.-ResponseClass',
+            contextKey: 'EWS.MovedItems',
```

Comment:
Regarding context - Optimal result is to also update exiting context object as well
('EWS.Items' & 'EWS.SearchItems' with existing ID)

@aviadl what do you think? we can skip this and if user needs it - he/she can search again